pkg/api/server/register_images.go

```diff
@@ -238,10 +242,9 @@ func (s *APIServer) registerImagesHandlers(r *mux.Router) error {
 	//      $ref: "#/responses/InternalError"
 	r.Handle(VersionedPath("/images/{name:..*}/json"), APIHandler(s.Context, generic.GetImage))
 	// swagger:operation POST /images/{nameOrID}/tag images tagImage
-	//
-	// Tag an image
-	//
 	// ---
+	// summary: Tag an image
+	// description: Tag an image so that it becomes part of a repository.
```

Comment:
Nit: "so that it becomes part of a repository" is not entirely true in the libpod world. `podman tag $ID foo` resolves to `localhost/foo` which strictly speaking isn't a real repository.